File: El Mamoun/Lv 1/Problem 37.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

/*
Problem 37: Write a program to read numbers from user and sum them, keep reading until the user enters -99 then print 
the Sum on screen.
*/
namespace Problem_37
{
    internal class Program
    {
        static float ReadNumber()
        {
            float Number = 0;
            Console.WriteLine("Please Enter a Number?: ");
            Number = Convert.ToSingle(Console.ReadLine());

            return Number;
        }
        static float SumNumUntil_99()
        {
            float Number = 0;
            float Sum = 0;
            int Counter = 1;
            do
            {
                Console.WriteLine("Please Enter Number" + Counter);
                Number = Convert.ToInt32(Console.ReadLine());
                if(Number==-99)
                {
                    break;
                }

                Counter++;
                Sum += Number;
                
            } while (Number != -99);

            return Sum;
        }


        static void Main(string[] args)
        {

            Console.WriteLine(SumNumUntil_99());


            Console.ReadKey();
        }
    }
}

File: El Mamoun/Lv 1/Problem 37.cpp
#include<iostream>
#include<string>
using namespace std;

/*
Problem 37: Write a program to read numbers from user and sum them, keep reading until the user enters -99 then print
the Sum on screen.
*/

float ReadNumber(string Message)
{
	float number = 0;

	cout << Message << endl;
	cin >> number;

	return number;
}

float SumUntil_99()
{
	int Sum=0, Counter=1, Number;

	do
	{
		Number = ReadNumber("Please Enter Number " + to_string(Counter));

		if (Number == -99)
		{
			break;
		}

		Counter++;
		Sum += Number;

	} while (Number != -99);

	return Sum;
}


int main()
{
	cout << "Result = " << SumUntil_99();

	return 0;
}